### ISSUE TITLE
Trust statistics wrong

### ISSUE THREAD

**Issue by Nubro01** [Tribler/tribler#5247]
#### Please, fill all relevant items: ####

- [x] I have read CONTRIBUTING.rst

- [x] I have tried with the latest pre-release version and I still can reproduce the issue.

##### Tribler version/branch+revision: #####
7.5 rc1
##### Operating system and version: #####
Windows 10 18363.720
##### Steps to reproduce the behavior: #####
Run tribler 7.x
##### Expected behavior: #####
While seedng Token balance while seeding then Given should increase.
##### Actual behavior: #####
When NOT downloading and ONLY Seeding Token balance should decrease not increase.
Started and giving was on 1882 and taken on 2489
Seeded large file of 3.8 gig after a while status was on given 1882 and taken on 2535 while I was ONLY giving not taken so.......

**Comment by ichorid** [comment]
There is a known "token" leak when you're seeding through the exit nodes to non-Tribler users. You're still technically "paying" the exit node for bittorrent protocol acknowledgements packets in this case. We have not decided yet how to tackle this problem. One solution would be to only allow hidden seeding to Tribler users, but that requires fixing hidden services. In 7.5, bootstrap file is shared through hidden services system to test how well it works.

In short, solving this is harder than it seems, but we're moving there. Though, the fix is not going to be there in 7.5.0.

**Comment by Nubro01** [comment]
If I go to the Tribler Debug pane and look at the IPv8 statistics then that shows;
total_up 12.44 MB
total_down 10.69 MB
How come these reflect the counts better?

**Comment by devos50** [comment]
I think we have already plenty of issues around the wrong token balance, so I'm closing this one. We acknowledge that the issue can be annoying and misleading, but solving it is indeed non-trivial.
